proto/hummock.proto

```diff
@@ -543,7 +543,7 @@ message VacuumTask {
 
 // Scan object store to get candidate orphan SSTs.
 message FullScanTask {
-  uint64 sst_retention_time_sec = 1;
+  uint64 sst_retention_watermark = 1;
```

Comment:
The watermark is meta node's now - SST retention sec.
Previously it's calculated in compute node, which relies on compute node's local clock.

Reply:
Just a note: the reason why this change is compatible is because the previous `sst_retention_time_sec` must be way smaller than `sst_retention_watermark`. That means:
- If meta is upgraded before compactor, compactor will be less aggressive to delete objects because `now - sst_retention_watermark` (using old logic to interpret the new field) is way smaller.
- If compactor is upgraded before meta, compactor will also be less aggressive to delete objects because `sst_retention_time_sec` (using new logic to interpret the old field) is way smaller